src/mlpack/core/cv/metrics/average_strategy.hpp

```diff
@@ -21,6 +21,13 @@ namespace cv {
  *
  * The "Binary" strategy means binary classification is going to be used, and
  * there is no need to average.
+ *
+ * The "Micro" strategy is used for multiclass classification considering
+ * TP(True Positive), TN(True Negative) and FN(False Negative) all at once
+ * for evaluating both precision and recall.
+ *
+ * The "Macro" strategy is also used for multiclass classification but by
+ * taking the average of individually calculated precision and recall.
```

Comment:
I'm assuming by `individually calculated` you mean that the precision and recall is individually computed for each class, and then averaged.  But the problem here is that `AverageStrategy` has nothing to do with precision and recall, and actually may not even be used by the different loss strategies.  So linking this documentation to a specific metric is something we should not do.

Reply:
@rcurtin I corrected the docs for micro and macro hope this time it's more robust and correctly defined.
I also removed the (be careful !) from imputer.hpp as it seems irrelevant and missguiding.